### DISCUSSION TITLE
Use Modrinth for publishing new builds

### DISCUSSION THREAD

**Discussion by MagmaSlime123** [eccentricdevotion/TARDIS#643] [Ideas]
### Describe the feature request

Modrinth is pretty much a WIP alt. for CurseForge. This could benefit TARDIS and its addons, and would give them a bit more attention.

It also has a much better API which can allow automation of new TARDIS builds being published onto the site after approval, like Jenkins. No need for manual uploads to Bukkit or Spigot, just let Modrinth's API and GitHub do the magic for you.

https://modrinth.com/

If you haven't heard/seen the recent drama going on about CurseForge's new and VERY restrictive API, then I'd recommend joining the Modrinth discord.

**Comment by Chew** [comment]
Commenting on this since jenkins reminded me. I would like to see TARDIS/Chunkgenerator on Modrinth, even if manually. Could have 1 featured version for TARDIS and 1 for ChunkGenerator? I at least want to somehow make it the Jenkins build to have all that data.

**Reply by MagmaSlime123** [reply]
Agreed. Could use GitHub actions instead of Jenkins probably.

**Reply by Chew** [reply]
We would lose the build number and version checking then, unless we implemented a better way

**Reply by MagmaSlime123** [reply]
Ah.

**Comment by MagmaSlime123** [comment]
See Discord.

**Reply by eccentricdevotion** [reply]
Don't use Discord, so will need to discuss here if any actions are to be taken...

**Comment by eccentricdevotion** [comment]
https://modrinth.com/user/eccentric

**Reply by MagmaSlime123** [reply]
Neat! You did forget the resourcepacks along with TWA and VortexManipulator though.

**Reply by eccentricdevotion** [reply]
Resource packs are there, they didn't get approved initially as the description was too short...

**Reply by MagmaSlime123** [reply]
Ah. What about TWA and TVM?
